core/src/scss/components/button/_button--big.scss

```diff
@@ -1,7 +1,22 @@
 @charset "UTF-8";
 
 // Big button.
-.su-button--big,
-.su-button--big.su-link {
-  @include button-big;
+.su-button {
+  &--big {
+    &,
+    &.su-link {
+      @include button-big;
+    }
+  }
+}
+
+button {
+  &.su-button {
+    &--big {
+      &,
+      &.su-link {
```

Comment:
@cjwest Do please double check, but I believe we have `<a>` with the `.su-button` and `.su-link` classes, but I don't think we have `<button>` with `.su-link` class.